jvm-runtime/ftl-runtime/common/deployment/src/main/java/xyz/block/ftl/deployment/DatabaseConfigSource.java

```diff
@@ -0,0 +1,56 @@
+package xyz.block.ftl.deployment;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+public class DatabaseConfigSource implements ConfigSource {
+
+    private final Map<String, String> properties;
+
+    public DatabaseConfigSource() {
+        properties = new HashMap<>();
+        var ds = Thread.currentThread().getContextClassLoader().getResourceAsStream("META-INF/ftl-sql-databases.txt");
+        if (ds == null) {
+            return;
+        }
+        try (var in = ds) {
+            Properties p = new Properties();
+            p.load(in);
+            for (var name : p.stringPropertyNames()) {
+                properties.put("quarkus.datasource." + name + ".db-kind", p.getProperty(name));
+            }
+            if (properties.size() == 1) {
+                try {
+                    // Temporary implicit support for hibernate orm
+                    Thread.currentThread().getContextClassLoader()
```

Comment:
🧐

Reply:
I will remove this once we have SQL verbs.